### ISSUE TITLE
Discovery Plus - Unable to download subs.

### ISSUE THREAD

**Issue by jack-sparrow1** [yt-dlp/yt-dlp#1737]
### Checklist

- [X] I'm asking a question and not reporting a bug/feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones

### Question

Hi, I'm trying to download a few series from Discovery Plus, and while the video gets downloaded perfectly, I'm unable to download the subs along with it.

I'm using this code to download it

`yt-dlp --cookies discovery_cookies.txt https://www.discoveryplus.in/videos/extreme-homes/red-rock-concrete-log-cabin?seasonId=1&type=EPISODE`

This downloads the episode, but I get the below warning when the download starts
 `Ignoring subtitle tracks found in the HLS manifest`

I've also tried using `--write-auto-subs` but even that does not work. Is there something that I'm missing here?

**Comment by Ashish0804** [comment]
This is the standard message when subs are in the m3u8 manifest but not extracted. 

DSCP+in needs an overhaul anyways, I'll get to it after exams (mid-end December).

Tl;dr: You can't download subs for discoveryplus.in atm

**Comment by pukkandan** [comment]
So, what is your question? Just because the `question` template is less work to fill up than others, don't use it to report bugs. If the subs don't show up in `--list-subs`, you should have opened a extractor-bug-report/site-feature-request

**Comment by Ashish0804** [comment]
Closing this in favor of https://github.com/yt-dlp/yt-dlp/issues/1742
